src/NuGet.Clients/VisualStudio/Extensibility/CredentialRequestType.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace NuGet.VisualStudio
+{
+    /// <summary>
+    /// The type of credential request being made.
+    /// </summary>
+    public enum CredentialRequestType
```

Comment:
Do you mean smth. like `Default`? If so the normal should be equal to `Unauthorized` for backcompat.